gammapy/time/plot.py

```diff
@@ -2,31 +2,126 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-__all__ = ['plot_time_difference_distribution',
+from ..datasets import fetch_fermi_catalog
+
+__all__ = ['plot_light_curve',
            ]
 
 
-def plot_time_difference_distribution(time, ax=None):
```

Comment:
The diff on Github looks complex because you've removed the `plot_time_difference_distribution` function.
Can you please put that back in, I wanted to implement it eventually.

The easiest might be to copy & paste it from here:
https://github.com/gammapy/gammapy/blob/master/gammapy/time/plot.py#L9